docs/introduction/StarRocks_intro.md

```diff
@@ -62,41 +62,41 @@ Join our [Slack channel](https://join.slack.com/t/starrocks/shared_invite/zt-z5z
 <NavBoxPart title="Querying​">
 <NavBoxPartItem title="Query acceleration">
 
-- [Cost-based optimizer](../using_starrocks/Cost_based_optimizer.md)
-- [Synchronous materialized view](../using_starrocks/Materialized_view-single_table.md)
-- [Asynchronous materialized views](../using_starrocks/Materialized_view.md)
-- [Colocate Join](../using_starrocks/Colocate_join.md)
-- [Query cache](../using_starrocks/query_cache.md)
+- [Cost-based optimizer](../using_starrocks/Cost_based_optimizer)
+- [Synchronous materialized view](../using_starrocks/Materialized_view-single_table)
+- [Asynchronous materialized views](../using_starrocks/Materialized_view)
```

Comment:
the singular form "view" is used for sync mv, why the plural form "views" is used for async mv